client/src/action-creators/cart-action-creator.js

```diff
@@ -0,0 +1,25 @@
+import * as cartActions from '../actions/cart-actions';
```

Comment:
Think about creating Store **folder** for all redux's stuff:
Store
   -cart
       --cart-actions
       --cart-reducer